Formula/cpufetch.rb

```diff
@@ -0,0 +1,18 @@
+class Cpufetch < Formula
+  desc "Simple yet fancy CPU architecture fetching tool"
+  homepage "https://github.com/Dr-Noob/cpufetch"
+  url "https://github.com/Dr-Noob/cpufetch/archive/v1.00.tar.gz"
+  sha256 "2254c2578435cc35c4d325b25fdff4c4b681de92cbce9a7a36e58ad58a3d9173"
+  license "MIT"
+  head "https://github.com/Dr-Noob/cpufetch.git", branch: "master"
+
+  def install
+    system "make"
+    bin.install "cpufetch"
+    man1.install "cpufetch.1"
```

Comment:
```suggestion
    system "make", "PREFIX=#{prefix}", "install"
```

Reply:
I intentionally wrote it this way because the Makefile is not stable at the moment (I recently requested upstream to change it).